src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -137,8 +146,15 @@ const MOCK_API_RESPONSE_DELAY = 5000;
     chiColumnsReset: e => {
       console.log('chiColumnsReset', e);
     },
-    chiSelectedRowsChange: e => {
-      console.log('chiRowSelect', e);
+    chiSelectedRowsChange(data) {
+      this.$data.bulkActionConfig = {
```

Comment:
Are we changing the config based on the captured event?

Reply:
Yes, we are changing based on it.